internal/db/notification_test.go

```diff
@@ -0,0 +1,109 @@
+package db
+
+import (
+	goContext "context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/slovak-egov/einvoice/internal/entity"
+)
+
+func testInvoiceStatus(t *testing.T, id int, status string) {
+	inv, err := connector.GetInvoice(ctx, id)
+	if err != nil {
+		t.Fatal(err)
+	}
+	assert.Equal(t, status, inv.NotificationsStatus)
+}
+
+func TestGetAndUpdateNotNotifiedInvoices(t *testing.T) {
```

Comment:
nice test 👍 
also pls add some comments so it is easy to see what we test here.
Either one explaining comment above test, or few smaller comments during test